cw/spark.py

```diff
@@ -1,176 +1,115 @@
 #!/usr/bin/env python
-import ast
 import base64
 import ccd
 import hashlib
-import os
+import math
 import numpy as np
-import sys
 import requests
+import xarray as xr
+import pandas as pd
 import cw
-from glob import glob
 from datetime import datetime
 
+
 class SparkException(Exception):
     pass
 
+
 class Spark(object):
     def __init__(self, config):
         self.config = config
```

Comment:
use this as the sole entry point for configuration data into your class.  Not ideal IMO because... "class", but it's pretty clean as long as we restrict the data flow paths and don't mutate state.